src/components/TextArea/AutoFocusTextArea.js

```diff
@@ -30,6 +30,7 @@ export default class AutoFocusTextArea extends React.Component {
         style={{...styles.input, ...this.props.style}}
         textAlignVertical={'top'}
         placeholder={this.props.placeholder}
+        keyboardAppearance="dark"
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `keyboardAppearance="dark"` prop has been added to the TextInput component. This changes the keyboard appearance to dark mode on iOS devices. However, this prop has no effect on Android devices. If you want to support both platforms uniformly, consider implementing a platform-specific solution.
```diff
-        keyboardAppearance="dark"
+        keyboardAppearance={Platform.OS === 'ios' ? "dark" : "default"}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->